### ISSUE TITLE
Buildout using the live.cfg script is broken

### ISSUE THREAD

**Issue by andreasma** [kitconcept/buildout#3]
I run the buildout first with make on Debian Buster and I got a working instance. The instance was created using Python 3.7 and Plone 5.2.x.
Then I tried to run the buildout again using the live.cfg script, but it didn't work. The buidlout broke in the varnish section. I got the following error message:

Got collective.recipe.backup 4.1.1.
Getting distribution for 'zc.recipe.cmmi'.
Got zc.recipe.cmmi 3.0.0.
Getting distribution for 'collective.recipe.template==2.1'.
WARNING: The easy_install command is deprecated and will be removed in a future version.
Got collective.recipe.template 2.1.
Getting distribution for 'plone.recipe.varnish'.
WARNING: The easy_install command is deprecated and will be removed in a future version.
warning: no previously-included files matching '*.pyc' found anywhere in distribution
Got plone.recipe.varnish 2.3.0.
While:
  Installing.
  Getting section varnish.
  Initializing section varnish.
  Loading zc.buildout recipe entry plone.recipe.varnish:default.

An internal error occurred due to a bug in either zc.buildout or in a
recipe being used:
Traceback (most recent call last):
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 2174, in main
    getattr(buildout, command)(args)
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 701, in install
    [self[part]['recipe'] for part in install_parts]
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 701, in <listcomp>
    [self[part]['recipe'] for part in install_parts]
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 1324, in __getitem__
    options._initialize()
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 1432, in _initialize
    self.initialize()
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 1438, in initialize
    recipe_class = _install_and_load(reqs, 'zc.buildout', entry, buildout)
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/zc/buildout/buildout.py", line 1393, in _install_and_load
    req.project_name, group, entry)
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/andreas/plone/githubrepos/kitconcept_buildout/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2851, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('zc.buildout', 'default') not found

**Comment by tisto** [comment]
@andreasma yeah, you are correct. This was planned to be used but we never came around sharing our prod setups because they are usually too complex and customer-specific. What are you looking for in particular? Maybe we can help...

**Comment by andreasma** [comment]
I migrated the extensions.libreoffice.org github repo to Python 3. The buildout worked with the staging.cfg script, but I run into errors with the live.cfg script. The issues are especially from haproxy  and varnish. That are areas where I didn't got my fingers dirty yet. I'm keen to learn more about this tools and their configuration / deployment and find a solution for the migration.
This was the reason why I tried your buildout and hoped to get some knowledge about best practice.

**Comment by andreasma** [comment]
I did some editing in the buildout environment of extensions.libreoffice.org these days and managed to fix the building of varnish within a new buildout script https://github.com/andreasma/extensions.libreoffice.org/blob/master/live2.cfg
I use in the script the plone.recipe.varnish from the Github repository and Varnish 6.0.6.
I had to put the haproxy part away and used for testing an edited vcl script of this buildout. But the supervisor executor and varnish stated that varnish could not start (stable), because the port of varnishd is already in use. A look with netstat showed me, that varnishd is up on that port and I could connect via localhost and varnishd's port to my Plone site.
The error message inside the varnish-log seemed to be strange.
